docs/best-practices.md

```diff
@@ -45,15 +45,31 @@ to a shared library function. You can introduce it in one of these two places:
   [leapp-repository on GitHub](https://github.com/oamg/leapp-repository).
 
 
-## Discover Leapp Standard Library functions
+## Discover standard library functions
 
 Before implementing functionality for your actor, browse through the available functionality provided by:
 - the [Leapp Standard Library](https://github.com/oamg/leapp/tree/master/leapp/libraries/stdlib/),
-- the shared library of your Leapp repository (`<Leapp repository>/libraries` folder),
+- the shared library of your Leapp repository (`<Leapp repository>/libraries` folder).
 
 These libraries contain functions that may satisfy your needs. Using them can save you time, lower code complexity and
 help avoiding duplicate code. Improvement proposals for the library functions are welcome.
 
+
+## Prefer using stdlib functions over shell commands
+
+Sources of external functionality to be used in your actor in order of preference:
+1. the [Leapp Standard Library](https://github.com/oamg/leapp/tree/master/leapp/libraries/stdlib/)
+2. the [Python Standard Library](https://docs.python.org/3/library/index.html)
+3. shell commands
```

Comment:
What about full path to the shell command - use `cp` or `/bin/cp`?

Reply:
I would say using full path is safer as no alias is executed instead of the expected command, and we don't need to rely on the content of PATH. On the other hand, and I'm not sure about this, the path may be different than expected when booted in the initramfs after doing some chroot/pivot magic. But that would get revealed during testing of the actor.

I'll mention it here, even though this seems more like a rather small detail IMO.